src/js/lib/key-derivation.js

```diff
@@ -1,11 +1,19 @@
+import bitcoin from 'bitcoinjs-lib'
+import bip39 from 'bip39'
+
 /* @summary - Generates a keypair based on provided entropy
  *
  * @param {Mnemonic} seed - a Bitcore-Mnemonic generated from hashed entropy.
  * @returns {HDPrivateKey} - an HDPrivateKey instance containing a keypair.
```

Comment:
We need to update comment block - now it's bitcore-lib specific

Reply:
Good point, can you perhaps do that quickly?